deeds-dapp-service/src/main/java/io/meeds/deeds/service/MeedTokenMetricService.java

```diff
@@ -156,6 +175,42 @@ public Map<String, BigDecimal> getLockedBalances() {
     return lockedBalances;
   }
 
+  /**
+   * @return {@link BigDecimal} . This will return the Makert Capitalization value of the Meeds Token
+   * throughout the formula of MarketCap = TotalSupply * Meeds price
+   */
+  public BigDecimal getMarketCapitalization() {
```

Comment:
You should add this operation in `ExchangeService` to have a single reference where to get the Meed USD price

Reply:
I made two seperate function from getMarketCapitalization() -> getMarketCapitalization + getMeedUsdPrice() inorder to inject the price in the metric seperately ! 
Still need the change you proposed ?